Refs #35591 -- Emphasized that runserver is not suitable for production. (#18457)
django/django: docs/ref/django-admin.txt

This server uses the WSGI application object specified by the
:setting:`WSGI_APPLICATION` setting.

.. warning:: DO NOT USE THIS SERVER IN A PRODUCTION SETTING.

>     This server has not gone through security audits or performance tests, and
>     hence is unsuitable for production. We're in the business of making web
>     frameworks, not web servers, so improving this server to be able to handle a
>     production environment is outside the scope of Django.
> 

Reviewer: ```suggestion
    This lightweight development server has not gone through security audits or
    performance tests, hence is unsuitable for production. Making this server
    able to handle a production environment is outside the scope of Django.
```
I don't love the original "we're in the business of" sentence. I quite like how above it is described as "lightweight development" and so am adding this in a few places to emphasize it's purpose
